dist/api_data.py

```diff
@@ -271,7 +271,9 @@ def __cmp__(self, other):
         leaf pages''',
         min=0),
     Config('split_deepen_min_child', '0', r'''
-        minimum entries in a page to consider deepening the tree''',
+        minimum entries in a page to consider deepening the tree. Once
+        there are more than this many entries pages will be immediately
+        considered for splitting''',
```

Comment:
@agorrod, I'd rephrase this to talk about deepening the tree, not `considered for splitting`.